controllers/cloud.redhat.com/providers/dependencies/impl.go

```diff
@@ -24,16 +24,25 @@ func (dep *dependenciesProvider) makeDependencies(app *crd.ClowdApp) error {
 	// Process self endpoints
 	appMap := map[string]crd.ClowdApp{app.Name: *app}
 	appRefMap := map[string]crd.ClowdAppRef{} // empty since we're only processing self
+
+	// Avoid mapping ports when TLS is false
+	tlsPort := int32(0)
+	tlsPrivPort := int32(0)
+	if dep.Env.Spec.Providers.Web.TLS.Enabled {
```

Comment:
TLS can be turned on/off for a ClowdApp in more than 1 place now (see the function `isPublicTLSEnabled` and `isPrivateTLSEnabled`, you probably want to use those here now instead of only checking the ClowdEnvironment)

Reply:
Yeah, I saw that, but I don't want to overengineering so much this fix. Also I didn't sure the order of this, I mean, if you have the tls enable in env but false in clowdapp, it should be enable or disable? I can add something more robust in term of reliability but need to know the order to apply the different conditions

Reply:
If TLS is enabled in env but false in ClowdApp, it will be disabled on the app

The `isPublicTLSEnabled` and `isPrivateTLSEnabled` checks both the env and the app